Dockerfile.apk

```diff
@@ -3,10 +3,10 @@ FROM alpine:3.17
 
 LABEL maintainer="Kong Docker Maintainers <docker@konghq.com> (@team-gateway-bot)"
 
-ARG KONG_VERSION=3.1.1
+ARG KONG_VERSION=3.2.1
 ENV KONG_VERSION $KONG_VERSION
 
-ARG KONG_AMD64_SHA="9194977814e91b6997ce4795957d9b4bcec9d220667e01a08ade7ac64704f57c"
+ARG KONG_AMD64_SHA="bf9a3d6b9b7fce1abfb1119e653e1609524a93ab34fc96af1f9b7428dac1ddf0"
 ARG KONG_ARM64_SHA="92d98307a3193602bbe4dbed2d099cde5404a8acd09d53f0b9a9f2bcf6038500"
```

Comment:
Since we are no longer making an arm64 package for Alpine, we may just want to remove its checksum and make `KONG_AMD64_SHA` just `KONG_SHA256`.